.travis.yml

```diff
@@ -1,10 +1,21 @@
 sudo: required
 dist: trusty
+
 language: c
+
 compiler:
   - gcc
   - clang
 
+env:
+  - RUN_TESTS=false
+  - RUN_TESTS=true
+
+matrix:
+  allow_failures:
+      - env: RUN_TESTS=true
```

Comment:
If you are going to just ignore tests failing, you might as well remove the testing step from the CI entirely.